aten/src/ATen/native/native_functions.yaml

```diff
@@ -9833,6 +9833,10 @@
   python_module: special
   variants: function
 
+- func: special_log_softmax(Tensor self, int dim, ScalarType? dtype=None) -> Tensor
```

Comment:
Should we make dim and dtype kwarg-only?

Reply:
Making `dtype` kwarg-only makes sense. 
However `dim` is a required argument so I don't think it should be kwarg-only

Reply:
Have updated the signature to `special_log_softmax(Tensor self, int dim, *, ScalarType? dtype=None) -> Tensor`